packages/config/config/devices/0x003b/s-6500f.json

```
@@ -58,6 +58,7 @@
 		{
 			"#": "6",
 			"label": "User Slots",
+			"description": "This value represents a bitmask with 30 bits. Each bit represents a user slot. A value of 1 means that the slot is occupied.",
```

```suggestion
			"description": "This value represents a bitmask with 10 bits. Each bit represents a user slot. A value of 1 means that the slot is occupied.",
```
If the maxValue is to be trusted